src/frameworks/index.ts

```diff
@@ -47,6 +48,7 @@ export const frameworks: Framework[] = [
   new EmberFramework(),
   new I18nextFramework(),
   new ReactI18nextFramework(),
+  new ShopifyI18nextFramework(),
```

Comment:
When I tophatted this, I noticed that the React framework was being auto-set as the enabled framework before Shopify (in a non-monorepo setup). 

Since we'll be setting frameworks explicitly in the templates, this might not be a problem, but I wonder if this will cause the ReactI18next framework code to execute instead of the ShopifyI18next framework code if it isn't explicitly set.

```
🧩 Enabled frameworks: React I18next, Shopify I18next, General
```

When I tried moving `new ShopifyI18nextFramework(),` ahead of `new ReactI18nextFramework()`, that caused the ShopifyI18next framework to load first in the list:

```
🧩 Enabled frameworks: Shopify I18next, React I18next, General
```